nncf/torch/layers.py

```diff
@@ -96,6 +96,7 @@ def _custom_forward_fn(self, input_):
         proxy_weight = self.weight
         return self._conv_forward(input_, proxy_weight, proxy_padding_value)
 
+    # pylint: disable=W0237
```

Comment:
I think the `_conv_forward` function shouldn't have the same name as in the original implementation of Conv2d.
The warning should be resolved by simply renaming this method to something different, e.g. `_conv_forward_proxy`

Reply:
Ok, i'll change the name of this function